plugins/postcss-color-function/README.md

```diff
@@ -90,7 +90,7 @@ postcssColorFunction({ preserve: true })
 	--a-color: rgb(164, 49, 43);
 }
 
-@supports (color: color(srgb 0 0 0)) {
+@supports (color: color(display-p3 0 0 0)) {
```

Comment:
Assuming this is intentional but looks off?

Reply:
Yes this is correct :)

It's more optimal to group the checks a bit instead of having a unique supports check per color space.